README.md

```diff
@@ -64,11 +64,46 @@ See [the wiki on github](https://github.com/01org/parameter-framework/wiki).
 
 ## Compiling
 
+### Dependencies
+
+In order to compile you'll need, at the very least:
+
+- CMake (v2.8.12 or later);
+- A C/C++ compiler supporting C++11;
+- libxml2 headers and libraries (Provided by the `libxml2-dev` on debian-based
+distributions);
+- Standalone ASIO (1.10.6 or later) (Provided by `libasio-dev` on debian-based
+distributions) ASIO is C++ header-only ASynchronous-IO library.
+
+If you want to compile the *Python bindings*, you'll also need:
+
+- SWIG 2.0 (A binding generator);
+- Python2.7 development environment (Provided by `python2.7-dev` on debian-based
+distributions)
+
+If you want to *compile and run the tests*, you'll also need:
+
+- Catch (Provided by `catch` on debian-based distributions). Catch is a
+single-header test framework - as such you may also download it directly
+[here](https://raw.githubusercontent.com/philsquared/Catch/master/single_include/catch.hpp);
+- Python2.7 (Provided by `python2.7` on debian-based distribution - it is
+preinstalled on most distributions).
+
+If you want to *build the code documentation*, you'll need `doxygen` and
+`graphviz`. This doc is already available to you - see the wiki.
+
+**To list all available configuration options, try** `cmake -L`.
+
+### How-To
+
+If you are already familiar with CMake, you know what to do.
+
 Run `cmake .` then `make`.  You may then install libraries, headers and
 binaries with `make install`.  By default, they are installed under
 `/usr/local` on unix OSes; if you want to install them under a custom
 directory, you may do so by passing it to the `cmake .` command; e.g.
 
+    # Always use absolute paths in CMake options
```

Comment:
Never mind, keep it like this. :+1
---------------------------------------------------------------------
Intel Corporation SAS (French simplified joint stock company)
Registered headquarters: "Les Montalets"- 2, rue de Paris, 
92196 Meudon Cedex, France
Registration Number:  302 456 199 R.C.S. NANTERRE
Capital: 4,572,000 Euros

This e-mail and any attachments may contain confidential material for
the sole use of the intended recipient(s). Any review or distribution
by others is strictly prohibited. If you are not the intended
recipient, please contact the sender and delete all copies.